.travis.yml

```diff
@@ -6,38 +6,17 @@ sudo: true # give us 7.5GB and >2 bursted cores.
 git:
   depth: false
 before_install:
-    - export PULUMI_ROOT=/opt/pulumi
     - git clone https://github.com/pulumi/scripts ${GOPATH}/src/github.com/pulumi/scripts
-    # Dep for Go dependency management.
-    - curl https://raw.githubusercontent.com/golang/dep/master/install.sh | sh
-    # Gometalinter for good Go linting/hygiene.
-    - go get -v github.com/alecthomas/gometalinter
-    - gometalinter --install
-    # Node.js 6.10.2 for all JavaScript code (to match AWS Lambda).
-    - nvm install v6.10.2
-    # Install Yarn as per https://yarnpkg.com/lang/en/docs/install-ci/#travis-tab.
-    - curl -o- -L https://yarnpkg.com/install.sh | bash -s -- --version 1.2.1
-    - export PATH=$HOME/.yarn/bin:$PATH
-    # Install Twine so that we can publish Pip packages.
-    - pip install --upgrade --user twine==1.9.1
-    # Set our token so we can publish npm packages if needed.
-    - if [ "${TRAVIS_SECURE_ENV_VARS:-}" = "true" ]; then echo "//registry.npmjs.org/:_authToken=\${NPM_TOKEN}" > ~/.npmrc; fi
-    # Install the AWS CLI so that we can publish the resulting release (if applicable) at the end.
-    - pip install --upgrade --user awscli
-    - if [ "${TRAVIS_SECURE_ENV_VARS:-}" = "true" ]; then source ${GOPATH}/src/github.com/pulumi/scripts/ci/keep-failed-tests.sh; fi
-    # Install Pulumi
-    - curl -L https://get.pulumi.com/ | bash -s -- --version 0.14.0-rc1
-    - export PATH=$HOME/.pulumi/bin:$PATH
-    - sudo apt-get install pandoc
+    - source ${GOPATH}/src/github.com/pulumi/scripts/ci/prepare-environment.sh
```

Comment:
Can this be `PULUMI_SCRIPTS` as well?

Reply:
prepare-environment.sh sets PULUMI_SCRIPTS so future invocations can use it, but it has to use the raw path itself.